agent/taskresource/asmsecret/asmsecret.go

```diff
@@ -315,7 +316,8 @@ func (secret *ASMSecretResource) retrieveASMSecretValue(apiSecret apicontainer.S
 
 	secretValue, err := asm.GetSecretFromASMWithInput(input, asmClient, jsonKey)
 	if err != nil {
-		errorEvents <- fmt.Errorf("fetching secret data from AWS Secrets Manager in region %s: %v", apiSecret.Region, err)
+		logger.Debug(fmt.Sprintf("fetching secret data from AWS Secrets Manager in region %s", apiSecret.Region))
```

Comment:
This log is printed after an attempt to fetch secret data was already finished and resulted in an error. Can we update the log message to reflect the current state more accurately? We could also add `err` to the log message.

Reply:
removed log